Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM ubuntu:xenial
```

Comment:
Why not just take an alpine-node@ `latest` or `LTS` image, which would then let you drop all the other dependencies?